keep-ui/app/(keep)/alerts/TitleAndFilters.tsx

```diff
@@ -35,6 +41,20 @@ export const TitleAndFilters = ({
     isFromCalendar: false,
   });
 
+  useEffect(() => {
+    if (onDateRangeChange) {
+      if (timeFrame.start && timeFrame.end) {
+        onDateRangeChange({
+          start: timeFrame.start,
+          end: timeFrame.end,
+        });
+        return;
+      }
+
+      onDateRangeChange(null);
```

Comment:
why's the `onDateRangeChange(null);`?

Reply:
To trigger CEL build in upstream component when no timeframe option exists. it means CEL will not contain lastReceived filter.